pkgs/by-name/ni/nist-feed/package.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, jq
+, killall
+, libnotify
+, curl
+, busybox
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "nist-feed";
+  version = "0-unstable-2024-01-20";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "NIST-Feed";
+    rev = "775bd871490b680784a1855cdc1d4958a83a7866";
+    hash = "sha256-OcVf766q7vELYkGOEzQMLS6zH8Nn96ibGP+6kizHN28=";
+  };
+
+  patches = [
+    ./cron.patch
```

Comment:
Maybe the name of this patch could be a bit more descriptive?

`remove-cronjob-instantiation-feature.patch` or something along those lines?